### PR TITLE
[CCTRI-2186] Add 404 error handling and warning to CTR_ENTITIES_LIMIT for Qualys

### PR DESCRIPTION
**Pull request by oshynk**

### CODE REVIEW

**Review by MaxMhonc — APPROVED** [review]

File: code/requirements.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ pyjwt[crypto]==2.0.1
 flake8==3.9.0
 coverage==5.2.1
 pytest==6.2.2
+Werkzeug==1.0.1
```
Comment (TarasMaliuchenko) [inline code/requirements.txt:9]: Where do we use this dependency?

Comment (oshynk) [reply code/requirements.txt:9]: We do not use it explicitly. It's used by Flask. But somehow not the latest version of Flask downloads the latest version of Werkzeug and that's why some tests fail, so we need to specify the older version in the requirements.

Comment (TarasMaliuchenko) [reply code/requirements.txt:9]: Maybe we need to migrate to Flask v2.0.0 ?

**Review by TarasMaliuchenko — APPROVED** [review]

File: code/requirements.txt (hunk 1 above)
Comment (oshynk) [reply code/requirements.txt:9]: Yes, we need to consider this. I'm just wondering if there is a possibility that a change of Flask major version can cause some issues? Do we need to just update it or some additional action points?

**Pull request merged**